deny.toml

```diff
@@ -0,0 +1,71 @@
+# cargo-deny configuration for enterprise compliance
+
+[licenses]
+# List of explicitly allowed licenses
```

Comment:
If we're talking "enterprise-grade" and licenses, the repo should probably be set up with a [REUSE](<https://reuse.software/>) lint too. I've found REUSE really useful for not letting myself forget to mark licenses on files.